api/src/main/resources/org/openmrs/api/db/hibernate/OrderGroup.hbm.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0"?>
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+
+-->
+<!DOCTYPE hibernate-mapping PUBLIC
+        "-//Hibernate/Hibernate Mapping DTD 3.1//EN"
+        "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+<hibernate-mapping package="org.openmrs">
+
+    <class name="OrderGroup" table="order_group" batch-size="25">
+
+        <cache usage="read-write"/>
+
+        <id name="orderGroupId" type="java.lang.Integer"
+            column="order_group_id"
+            unsaved-value="undefined">
+            <generator class="org.openmrs.api.db.hibernate.NativeIfNotAssignedIdentityGenerator" />
```

Comment:
By convention we use the plain native id generator, is their a compelling reason for being different here?